config/nav.yml

```diff
@@ -184,9 +184,10 @@ nav:
           - Broker types:
             - Available Broker types: eventing/brokers/broker-types/README.md
             # add default IMC broker page, page explaining broker types
+            - Multi-tenant channel based Broker: eventing/brokers/broker-types/mt-channel-based-broker/README.md
```

Comment:
@knative/eventing-wg-leads @matzew do we still refer to it as Multi-tenant channel based Broker? I vaguely remember something about this changing?

Reply:
I'm not aware of that, I'd migrate the title to "Channel based Broker" to make it more concise

Reply:
I think that might have been it 🙂 it's been a while...

Reply:
I am totally fine with renaming it. We just should make sure we don't confuse the users with the naming <-> `MTChannelBasedBroker`